wallet/src/wallet/mod.rs

```diff
@@ -182,6 +183,16 @@ pub enum WalletError {
     ReducedLookaheadSize(u32, u32),
     #[error("Wallet file {0} error: {1}")]
     WalletFileError(PathBuf, String),
+    #[error("Missing inputs signatures in a tx")]
+    InputsSignatureAreMissing,
+    #[error("Cannot use partially signed transaction in a decommission command")]
```

Comment:
We need to deliver clearer information to the (noob) user to understand what's going on here. Basically the problem here is that we're trying to decommission the pool, but the wallet doesn't have all the required keys to fully sign the transaction. We could simply opt out to changing this error message to something like "Failed to completely sign the decommission transaction. This wallet does not seem to have the decommission key. Consider using a decommission-request, and passing it to the wallet that has the decommission key".